net-p2p/bitcoind/bitcoind-0.14.2.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 2010-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+BITCOINCORE_COMMITHASH="fc61c8322bd7288f7546d18ad04c36c345be13cd"
+BITCOINCORE_LJR_DATE="20170618"
+BITCOINCORE_IUSE="bip148 examples knots no-bip148 test upnp +wallet zeromq"
+BITCOINCORE_POLICY_PATCHES="+rbf spamfilter"
+BITCOINCORE_NEED_LEVELDB=1
+BITCOINCORE_NEED_LIBSECP256K1=1
+inherit bash-completion-r1 bitcoincore user systemd
+
+DESCRIPTION="Original Bitcoin crypto-currency wallet for automated services"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-linux ~arm ~arm64 ~mips ~ppc ~x86 ~x86-linux"
+
+REQUIRED_USE="^^ ( bip148 no-bip148 )"
+
+pkg_setup() {
+	local UG='bitcoin'
+	enewgroup "${UG}"
+	enewuser "${UG}" -1 -1 /var/lib/bitcoin "${UG}"
+}
+
+src_prepare() {
+	sed -i 's/have bitcoind &&//;s/^\(complete -F _bitcoind bitcoind\) bitcoin-cli$/\1/' contrib/${PN}.bash-completion || die
```

Comment:
This can't really be upstreamed, since the file is simply in `contrib/` and not managed by the build system. It's done to satisfy Portage QA issues IIRC.